src/OpenLoco/Windows/TextInputWindow.cpp

```diff
@@ -243,6 +243,17 @@ namespace OpenLoco::Ui::Windows::TextInput
         position = { inputSession.xOffset, 1 };
         Gfx::drawStringLeft(*clipped, &position, Colour::black, StringIds::black_stringid, _commonFormatArgs);
 
+        position = { window.x + widget->left, window.y + widget->top - 10 };
```

Comment:
Do `Ui::Point(window.x + widget->left, window.y + widget->top - 10)` to take care of this